package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "q-templates-application",
-    "version": "4.0.2",
+    "version": "4.1.0",
```

Comment:
I'm wondering of this should be a patch bump? as you are just removing "(optional)" due to it now being a newly introduced bug maybe?

Same logic for the version within the `index.js` file too?